crest/Assets/Crest/Crest/Scripts/Shapes/FFT/ShapeFFT.cs

```diff
@@ -0,0 +1,422 @@
+﻿// Crest Ocean System
+
+// Copyright 2021 Wave Harmonic Ltd
+
+using UnityEngine;
+using UnityEngine.Rendering;
+using Crest.Spline;
+using UnityEngine.Experimental.Rendering;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Crest
+{
+    /// <summary>
+    /// FFT ocean wave shape
+    /// </summary>
+    [ExecuteAlways]
+    [AddComponentMenu(Internal.Constants.MENU_PREFIX_SCRIPTS + "Shape FFT")]
+    [HelpURL("https://crest.readthedocs.io/en/latest/user/wave-conditions.html#shapefft")]
```

Comment:
This should be (for versioning and RP selection):
```csharp
[HelpURL(Internal.Constants.HELP_URL_BASE_USER + "wave-conditions.html" + Internal.Constants.HELP_URL_RP + "#shapefft")]
```

Reply:
oops, thanks